bennu-io/src/main/java/org/fenixedu/bennu/io/api/FileStorageResource.java

```diff
@@ -71,40 +66,40 @@ public JsonElement all() {
     @Produces(MediaType.APPLICATION_JSON)
     public JsonObject fileCount() {
         accessControl(Group.managers());
-        JsonObject json = new JsonObject();
-        for (FileStorage store : FileSupport.getInstance().getFileStorageSet()) {
+        final JsonObject json = new JsonObject();
+        for (final FileStorage store : FileSupport.getInstance().getFileStorageSet()) {
             json.addProperty(store.getExternalId(), store.getFileSet().size());
         }
         return json;
     }
 
     @Atomic
-    private DomainStorage createDomainStorageService(String name) {
+    private DomainStorage createDomainStorageService(final String name) {
         return FileStorage.createNewDomainStorage(name);
     }
 
     @DELETE
     @Path("{oid}")
     @Produces(MediaType.APPLICATION_JSON)
-    public JsonElement delete(@PathParam("oid") String storageOid) {
+    public JsonElement delete(final @PathParam("oid") String storageOid) {
```

Comment:
I think you can change this to the domain object itself since there's a jaxrs converter for that I believe. It will make the code much simpler and easy to read. The same applies in other methods as well.